test/interactions/clickInteractionTests.ts

```diff
@@ -2,156 +2,186 @@
-        let callback1 = () => callback1WasCalled = true;
+        assert.isTrue(callback1.called, "Interaction should trigger the first callback");
+        assert.isTrue(callback2.called, "Interaction should trigger the second callback");
+      });
 
-        let callback2WasCalled = false;
-        let callback2 = () => callback2WasCalled = true;
+      it("can deregister a callback without affecting the other ones", () => {
+        const callback1 = makeClickCallback();
+        const callback2 = makeClickCallback();
 
         clickInteraction.onClick(callback1);
         clickInteraction.onClick(callback2);
-        TestMethods.triggerFakeMouseEvent("mousedown", component.content(), 0, 0);
-        TestMethods.triggerFakeMouseEvent("mouseup", component.content(), 0, 0);
-        assert.isTrue(callback1WasCalled, "Click interaction should trigger the first callback");
-        assert.isTrue(callback2WasCalled, "Click interaction should trigger the second callback");
-
-        callback1WasCalled = false;
-        callback2WasCalled = false;
         clickInteraction.offClick(callback1);
-        TestMethods.triggerFakeMouseEvent("mousedown", component.content(), 0, 0);
-        TestMethods.triggerFakeMouseEvent("mouseup", component.content(), 0, 0);
-        assert.isFalse(callback1WasCalled, "Callback1 should be disconnected from the click interaction");
-        assert.isTrue(callback2WasCalled, "Callback2 should still exist on the click interaction");
+        clickPoint(testPoint);
 
-        svg.remove();
+        assert.isFalse(callback1.called, "Callback1 should be disconnected from the click interaction");
+        assert.isTrue(callback2.called, "Callback2 should still exist on the click interaction");
       });
+    });
 
-      it("does not trigger callback when touch event is cancelled", () => {
-        let callbackCalled = false;
-        let callback = () => callbackCalled = true;
-        clickInteraction.onClick(callback);
-
-        TestMethods.triggerFakeTouchEvent("touchstart", component.content(), [{x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2}]);
-        TestMethods.triggerFakeTouchEvent("touchcancel", component.content(), [{x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2}]);
-        TestMethods.triggerFakeTouchEvent("touchend", component.content(), [{x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2}]);
-        assert.isFalse(callbackCalled, "callback not called since click was interrupted");
-
-        svg.remove();
+    [TestMethods.InteractionMode.Mouse, TestMethods.InteractionMode.Touch].forEach((mode) => {
+      describe(`invoking callbacks with ${TestMethods.InteractionMode[mode]} events`, () => {
+        let callback: ClickTestCallback;
+
+        beforeEach(() => {
+          callback = makeClickCallback();
+          clickInteraction.onClick(callback);
+        });
+
+        it("invokes onClick callback on single location click", () => {
+          const point = {x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2};
```

Comment:
these points are used over and over again
do you think it makes sense to create const variables in outer scope (halfPoint, quarterPoint, etc..)